.github/workflows/publish-helm.yaml

```diff
@@ -23,6 +23,7 @@ on:
 
 env:
   HUB: ghcr.io/apache/skywalking-kubernetes
+  VERSION: 0.0.0-${{ github.sha }}
```

Comment:
As a snapshot, could we have a version prefix?

Reply:
> As a snapshot, could we have a version prefix?

like this `-snapshot`?

Reply:
prefix, I mean, such as 4.5.0-xxxx

Reply:
> prefix, I mean, such as 4.5.0-xxxx

Sure. the rationale here I use 0.0.0-xxx is that, to mimic the same version style as our existing snapshot Docker images, whose version is simply `<sha>`, without a version prefix like `9.5.0-<sha>`, when we want to use a snapshot of a particular commit in the main branch, we don't have to find out what version is it currently, we can just copy the commit sha and use it `0.0.0-<sha>`.


> The `0.0.0-` prefix is just to adhere the version format of Helm Chart, it must be `\d+.\d+.\d-\w+` style.